### PR TITLE
feat(ymap): add calculate extents

### PR DESCRIPTION
**Pull request by PichotM**
This PR introduces extent calculation for ymaps on save and with a button.

I've tested my changes with:
- Custom ymaps
- Vanilla ymaps (drawable entities, occluders, cargens)

If all entities are loaded properly the extents are the same as in CodeWalker.
Did not run any tests though.

### CODE REVIEW

**Comment by ook3D** [comment]
nice work 😸

**Comment by ook3D** [comment]
closes https://github.com/Sollumz/Sollumz/issues/944

File: tools/ymaphelper.py [hunk 1]
```diff
@@ -89,3 +91,159 @@ def get_cargen_mesh() -> bpy.types.Mesh:
...
+
+class ExtentsData:
+    def __init__(self, lod_dist, bb_min, bb_max, bs_radius, scale):
+        self.lod_dist = lod_dist
+        self.bb_min = bb_min
+        self.bb_max = bb_max
+        self.bs_radius = bs_radius
+        self.scale = scale
+def get_extents_data(obj, entity_extents_data):
+    archetype_name = obj.name
```
Comment (alexguirre) [inline tools/ymaphelper.py:103]: Should use `remove_number_suffix` here.

**Comment by PichotM** [comment]
Updated

**Review by alexguirre — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**